src/transformers/trainer.py

```diff
@@ -1967,9 +1993,14 @@ def _load_optimizer_and_scheduler(self, checkpoint):
             # deepspeed loads optimizer/lr_scheduler together with the model in deepspeed_init
             return
 
-        if os.path.isfile(os.path.join(checkpoint, OPTIMIZER_NAME)) and os.path.isfile(
-            os.path.join(checkpoint, SCHEDULER_NAME)
-        ):
+        import glob
```

Comment:
`glob` is standard library, not an extra dep, so should be imported at the top of the file.

Reply:
Updated.